src/main/java/tds/exam/results/web/endpoints/TISCallbackController.java

```diff
@@ -0,0 +1,44 @@
+package tds.exam.results.web.endpoints;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpServletResponse;
+
+import java.util.UUID;
+
+import tds.exam.results.services.MessagingService;
+import tds.exam.results.tis.TISState;
+
+@RestController
+public class TISCallbackController {
+    private final MessagingService messagingService;
+
+    @Autowired
+    public TISCallbackController(final MessagingService messagingService) {
+        this.messagingService = messagingService;
+    }
+
+    @RequestMapping(value = "/tis", method= RequestMethod.POST, produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I based the TisState class and controller endpoint on:
https://github.com/SmarterApp/TDS_Student/blob/5a2f554b0d347fb78099f2ef279bd4f306f98dec/student/src/main/java/tds/student/web/handlers/TisHandler.java

Which uses this request object:
https://github.com/SmarterApp/TDS_Student/blob/5a2f554b0d347fb78099f2ef279bd4f306f98dec/student/src/main/java/tds/student/web/data/TisState.java

We provide the callback URL to TIS, but I don't think there would be any way to change the request type, route, or body without making changes to TIS.